PINCache/PINCache.m

```diff
@@ -74,6 +74,22 @@ + (instancetype)sharedCache
 
 #pragma mark - Public Asynchronous Methods -
 
+- (void)containsObjectForKey:(NSString *)key block:(PINCacheObjectContainmentBlock)block
+{
+    if (!key || !block) {
```

Comment:
The `!block` early-return condition makes sense, but I'm not sure about `!key` not executing the block. It's such an exceptional case that I don't think it's that important in practice, but I also think it's a stronger contract to say that the block is always invoked if one is provided.